ideainspect.groovy

```diff
@@ -242,6 +261,59 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println "#"
```

Comment:
On a side note: Due to the new process of restoring a backup copy after the run of the script which means that the user should never see a modified `idea.properties` files I'm fine with _not_ having those lines added to the properties file now.

I'd remove the output block on the cleanup, though.